package.json

```diff
@@ -9,6 +9,7 @@
     "@rails/webpacker": "5.4.3",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "bootstrap": "^5.1.3",
+    "jest": "^27.5.1",
```

Comment:
Ideally, this sort of package add would be in another commit and another PR, but since we're pressed for time this week, let's go with it!